src/CommonITILObject.php

```diff
@@ -11281,4 +11281,19 @@ public function canRequesterUpdateItem()
     {
         return true;
     }
+
+    /**
+     * Retrieve recipient user ID
+     *
+     * @return int|false
+     */
+    public function getRecipientUserID(): int|false
+    {
+        $users_id = $this->fields['users_id_recipient'] ?? 0;
+        $user = new User();
+        if ($users_id == 0 || !$user->getFromDB($users_id)) {
+            return false;
```

Comment:
For this method, it does not change much, but globally, it is easier to have `XXX|null` return values, as it eases the usage of null related operators (e.g. `$x = $item->getVal() ?? "default"`, `$item->getFoo()?->bar()`, ...).

```suggestion
            return null;
```